.github/workflows/full_build.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+      - static-code-analysis-tool
```

Comment:
Do we need to run CI in this branch

Reply:
Fixed in [3b047e1](https://github.com/ballerina-platform/static-code-analysis-tool/pull/3/commits/3b047e1a26221f0e7249d4d0fecd07cc94c608b7)